README.md

```diff
@@ -89,6 +89,11 @@ Various environment variables are set in the docker-compose file.
 
         docker-compose up
 
+   This will keep running continuously in your console window, so to execute further commands you'll need
+   to open a new console window or tab.
```

Comment:
Worth adding that if you don't want it to occupy the terminal (Console is actually a system event logging tool on Mac) you can specify the `-d` flag to run it in the background and that you can then stop the containers using `docker-compose down`?

Reply:
yep good idea, thanks